packages/orchestration/src/exos/stakingAccountKit.js

```diff
@@ -150,6 +208,23 @@ export const prepareStakingAccountKit = (baggage, makeRecorderKit, zcf) => {
             return this.facets.holder.delegate(validator, amount);
           }, 'Delegate');
         },
+        /**
+         * @param {CosmosValidatorAddress} srcValidator
+         * @param {CosmosValidatorAddress} dstValidator
+         * @param {AmountArg} amount
+         */
+        Redelegate(srcValidator, dstValidator, amount) {
```

Comment:
since all these makers just map the methods from the `holder`, I wonder if we should have some method on the holder to create invitation makers given a zcf

Reply:
I'm not clear on what you're suggesting.

Making another exoKit seems awkward.